packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -67,10 +68,11 @@ export class SelectionZone extends BaseComponent<ISelectionZoneProps, {}> {
 
   public componentDidMount() {
     let win = getWindow(this.refs.root);
+    let scrollElement = findScrollableParent(this.refs.root);
 
     // Track the latest modifier keys globally.
     this._events.on(win, 'keydown keyup', this._updateModifiers);
-    this._events.on(win, 'click', this._tryClearOnEmptyClick);
+    this._events.on(scrollElement, 'click', this._tryClearOnEmptyClick);
```

Comment:
@dzearing looking at this old code, `findScrollableParent` could be quite expensive (causes  the browser to recalc styles), wouldn't `this.ref.root` be enough here?